src/App.tsx

```diff
@@ -1,10 +1,70 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
 
-const USER_ID = 0;
+const USER_ID = 10565;
+
+enum ShowTodos {
+  All,
+  Active,
+  Completed,
+}
+
+const todosFromServer = client
+  .get<Todo[]>(`/todos?userId=${USER_ID}`);
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
```

Comment:
Better use null when there is no value, instead of empty string, as it is more explicit and non-ambigious indication that the value is empty.

Reply:
👍, I'm understood, thank you. Done.